LibsAndSamples.sln

```diff
@@ -104,7 +104,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "XFormsApp.UWP", "adal\devAp
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.MSAL.NET.Integration", "msal\tests\Test.MSAL.NET.Integration\Test.MSAL.NET.Integration.csproj", "{F126A28F-C3FB-4729-BBCB-77EF9CEB76EF}"
```

Comment:
shouldn't this be included inthe LibsNoSamples.sln as well?

Reply:
yes...thanks for the reminder.